apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -295,8 +306,11 @@ export const BaseTicketForm: React.FC<ITicketFormProps> = (props) => {
                                             ) : null
                                         }
                                         <Col span={24}>
-                                            <Form.Item name={'property'} label={AddressLabel}
-                                                rules={validations.property}>
+                                            <Form.Item
+                                                name={'property'}
+                                                label={AddressLabel}
+                                                rules={[...validations.property, { validator: addressValidation }]}
```

Comment:
maybe move to const array?

Reply:
@nomerdvadcatpyat could you please explain more detail? This array contain local function that in needed for local state forwarding to validator method. So it can't be moved outside component context

Reply:
@sitozzz i meant to take this out of the component render, before the `return` keyword

Reply:
@nomerdvadcatpyat ok, done this)